app/src/main/java/ch/epfl/skysync/MainActivity.kt

```diff
@@ -36,6 +36,10 @@ class MainActivity : ComponentActivity() {
           this.onSignInResult(res)
         }
 
-    setContent { SkySyncTheme { LoginScreen(signInLauncher = signInLauncher, user = user.value) } }
+    setContent { SkySyncTheme {
+      //LoginScreen(signInLauncher = signInLauncher, user = user.value)
+      showCalendarAvailabilities()
+      }
```

Comment:
You shouldn't modify the `MainActivity` file (it's great for testing but you shouldn't commit it)

Reply:
I agree it was a git mistake